ha/k8s_setup/const.py

```diff
@@ -25,6 +25,9 @@
 #Confstore delimiter
 _DELIM=">"
 
+# consule endpoint scheme: http
```

Comment:
Please change the name in comment as well.

Reply:
> Can you please add output of HA.conf, consul keys after mini provisioning and try to start health monitor service and see whether it starts or not!

done
-------------------------------------------------------------------------------------------------------------
the output of health monitor logs
-------------------------------------------------------------------------------------------------------------
2021-12-23 06:56:42 health_monitor [1095]: INFO [__init__] Health Monitor daemon initializations...
2021-12-23 06:56:42 health_monitor [1095]: INFO [run] Running the daemon for Health Monitor with PID 1095...
2021-12-23 06:56:42 health_monitor [1095]: INFO [__init__] MessageBus initialized as kafka
2021-12-23 07:02:58 health_monitor [1095]: INFO [evaluate] Evaluated action ['publish'] for key action/node/failed
2021-12-23 07:02:58 health_monitor [1095]: INFO [process_event] Evaluated {"event_id": "16402429775cdf7bcba2e94801b55b2187527cf922", "event_type": "failed", "severity": "error", "site_id": "1", "rack_id": "1", "cluster_id": "55097e16a7184ff080cbd6d261b27aab", "storageset_id": "3595344c9cc7469e8055d453e4fb738d", "node_id": "3595344c9cc7469e8055d453e4fb738d", "host_id": "3595344c9cc7469e8055d453e4fb738d", "resource_type": "node", "timestamp": "1640242977", "resource_id": "3595344c9cc7469e8055d453e4fb738d", "specific_info": "specific_info"} with action ['publish']
2021-12-23 07:02:58 health_monitor [1095]: INFO [act] Default action handler with Event: 16402429775cdf7bcba2e94801b55b2187527cf922 and actions : ['publish']
2021-12-23 07:02:59 health_monitor [1095]: INFO [publish] Sending action event {"version": "2.0", "event_type": "failed", "event_id": "16402429775cdf7bcba2e94801b55b2187527cf922", "resource_type": "node", "cluster_id": "55097e16a7184ff080cbd6d261b27aab", "site_id": "1", "rack_id": "1", "storageset_id": "3595344c9cc7469e8055d453e4fb738d", "node_id": "3595344c9cc7469e8055d453e4fb738d", "resource_id": "3595344c9cc7469e8055d453e4fb738d", "timestamp": "1640242977", "event_specific_info": "specific_info"} to component hare
2021-12-23 07:03:00 health_monitor [1095]: INFO [evaluate] Evaluated action [] for key action/rack/degraded
2021-12-23 07:03:00 health_monitor [1095]: INFO [evaluate] Evaluated action [] for key action/site/degraded
2021-12-23 07:03:00 health_monitor [1095]: INFO [evaluate] Evaluated action [] for key action/cluster/degraded
---------------------------------------------------------------------------------------------------------------------------

Reply:
updated ha monitor log in test scenario file which is attached in description https://github.com/Seagate/cortx-ha/files/7767395/test_scenarios_ha-monitor-logs_EOS-27071.txt